tsl/src/chunk_copy.c

```diff
@@ -624,13 +817,185 @@ chunk_copy(Oid chunk_relid, const char *src_node, const char *dst_node, bool del
 		ErrorData *edata;
 		MemoryContextSwitchTo(oldcontext);
 		edata = CopyErrorData();
-		edata->hint =
-			psprintf("chunk copy operation id: %d (%s).", cc.fd.id, NameStr(cc.fd.operation_id));
+		edata->hint = psprintf("chunk copy operation id: (%s).", NameStr(cc.fd.operation_id));
```

Comment:
Nit: Hints start with capitol letter. Also, should this perhaps be a detailed message rather than a hint? According to error style guide: "Use a hint message for suggestions about what to do to fix the problem, ..." and I don't think this message qualifies.

```suggestion
		edata->hint = psprintf("Chunk copy operation id: %s.", NameStr(cc.fd.operation_id));
```

Reply:
Done.